code/modules/surgery/organs/cybernetics/augments_legs.dm

```diff
@@ -0,0 +1,217 @@
+
+		owner.nutrition -= 5
+		owner.blood_volume--
+		owner.Jitter(1)
+		owner.Dizzy(1)
+
+		current_ticks_cooldown -= SSmobs.wait
+
+		return
+
+	if(owner.health < health_threshold)
+		current_ticks_cooldown = max_ticks_cooldown
+		on_effect()
+
+/obj/item/organ/cyberimp/leg/chemplant/emp_act(severity)
+	. = ..()
+	health_threshold += rand(-10,10)
+	current_ticks_cooldown = max_ticks_cooldown
+	on_effect()
+
+/obj/item/organ/cyberimp/leg/chemplant/proc/on_effect()
+	var/obj/effect/temp_visual/chempunk/punk = new /obj/effect/temp_visual/chempunk(get_turf(owner))
+	punk.color = implant_color
+	owner.reagents.add_reagent_list(reagent_list)
+
+	overlay = mutable_appearance('icons/effects/effects.dmi', "biogas",ABOVE_MOB_LAYER)
+	overlay.color = implant_color
+
+	RegisterSignal(owner,COMSIG_ATOM_UPDATE_OVERLAYS,.proc/update_owner_overlay)
+
+	addtimer(CALLBACK(src,.proc/remove_overlay),max_ticks_cooldown/2)
+
+	to_chat(owner,"<span class = 'notice'> You feel a sharp pain as the cocktail of chemicals is injected into your bloodstream!</span>")
+	return
+
+/obj/item/organ/cyberimp/leg/chemplant/proc/update_owner_overlay(atom/source, list/overlays)
+	SIGNAL_HANDLER
+
+	if(overlay)
+		overlays += overlay
+
+/obj/item/organ/cyberimp/leg/chemplant/proc/remove_overlay()
+	QDEL_NULL(overlay)
+
+	UnregisterSignal(owner,COMSIG_ATOM_UPDATE_OVERLAYS)
+
+/obj/effect/temp_visual/chempunk
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "chempunk"
+	pixel_x = -32 //So the big ol' 96x96 sprite shows up right
+	pixel_y = -32
+	layer = BELOW_MOB_LAYER
+	duration = 5
+
+/obj/item/organ/cyberimp/leg/chemplant/drugs
+	name = "deep-vein emergency morale rejuvenator"
+	desc = "Dangerous implant used by the syndicate to reinforce their assault forces that go on suicide missions."
+	implant_color = "#74942a"
+	encode_info = AUGMENT_SYNDICATE_LEVEL
+	reagent_list = list(/datum/reagent/determination = 5, /datum/reagent/drug/methamphetamine = 5 , /datum/reagent/medicine/atropine = 5)
+
+/obj/item/organ/cyberimp/leg/chemplant/emergency
+	name = "deep emergency chemical infuser"
+	desc = "Dangerous implant used by the syndicate to reinforce their assault forces that go on suicide missions."
+	implant_color = "#2a6194"
+	encode_info = AUGMENT_NT_HIGHLEVEL
+	reagent_list = list(/datum/reagent/medicine/atropine = 5, /datum/reagent/medicine/omnizine = 3 , /datum/reagent/medicine/leporazine = 3, /datum/reagent/medicine/c2/aiuri = 2, /datum/reagent/medicine/c2/libital = 2)
+
+/obj/item/organ/cyberimp/leg/chemplant/rage
+	name = "R.A.G.E. chemical system"
+	desc = "Extremely dangerous system that fills the user with a mix of potent drugs in dire situation."
+	implant_color = "#ce3914"
+	encode_info = AUGMENT_TG_LEVEL
+	reagent_list = list(/datum/reagent/determination = 2, /datum/reagent/medicine/c2/penthrite = 3 , /datum/reagent/drug/bath_salts = 5 , /datum/reagent/medicine/ephedrine = 5)
```

Comment:
1. this will braindeath you after two injections
2. the bathsalts (psychotic brawling) + ephedrine make this best for unarmed fighting. maybe put something in the desc to reflect this.

Reply:
i think the description conveys how dangerous this implant is already.